tests/cicd/main_build.yml

```diff
@@ -49,11 +49,16 @@ jobs:
     name: deepseismicagentpool
   steps:
   - bash: |
-      echo "Starting unit tests"
+      echo "Starting cv_lib unit tests"
       source activate seismic-interpretation
       pytest --durations=0 cv_lib/tests/
-      echo "Unit test job passed"
-    
+      echo "cv_lib unit test job passed"
+      echo ""
+      echo "Starting scripts unit tests"
+      source activate seismic-interpretation
+      pytest --durations=0 tests/
```

Comment:
Could you please place this in a separate build job? You can use cv_lib build job as an example, call this something like "job: common_tests" then include everything else: 
dependsOn: setup
timeoutInMinutes: 5
etc...